### PR TITLE
docs: describe your change...

### PR DESCRIPTION
**Pull request by Loli35**
Updated

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
none functional 

Issue Number: 1

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: aio/content/cli/index.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ Command syntax is shown as follows:
     Arguments are not prefixed.
     For example:
     <code-example language="bash">
-        ng build my-app -c production
+        Run build my-app -c production
```
Comment (mhevery) [inline aio/content/cli/index.md]: I don't believe that there is a `Run` command. Why is your motivation to change this from `ng`?

File: aio/content/cli/index.md [hunk 2]
```diff
@@ -106,12 +106,12 @@ Options that specify files can be given as absolute paths, or as paths relative
...
 ### Schematics
 
-The [ng generate](cli/generate) and  [ng add](cli/add) commands take as an argument the artifact or library to be generated or added to the current project.
+The [run generate](cli/generate) and  [run< add](cli/add) commands take as an argument the artifact or library to be generated or added to the current project.
 In addition to any general options, each artifact or library defines its own options in a *schematic*.
 Schematic options are supplied to the command in the same format as immediate command options.
 
 
-### Building with Bazel
+### Building with git
```
Comment (mhevery) [inline aio/content/cli/index.md]: Why are we changing instructions from Bazel to git?

**Review by mhevery — CHANGES_REQUESTED** [review]

**Comment by kapunahelewong** [comment]
@Loli35, do you have any questions with the process? Experimenting is totally fine - you can say `[WIP]` in the title of the PR or even state that you're just trying stuff out. If you need any help, just let me know.

**Comment by AndrewKushnir** [comment]
Closing this PR since it conflicts with the code in the main branch and there was no recent activity. Thank you.

**Pull request closed without merging**